src/seedu/addressbook/data/person/Address.java

```diff
@@ -38,19 +58,31 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return value;
+        StringBuilder result = new StringBuilder();
+        result.append(this.block);
+        result.append(", ");
+        result.append(this.street);
+        result.append(", ");
+        result.append(this.unit);
+        result.append(", ");
+        result.append(this.postalCode);
```

Comment:
Nice use of `StringBuilder` =). Perhaps we can improve this by chaining the method calls?

Perhaps this?

``` java
    return new StringBuilder().append(this.block).append(", ").append(....)
                ....
                .toString();
```